etc/kayobe/stackhpc-compute.yml

```diff
@@ -101,3 +101,15 @@ stackhpc_gpu_data:
     vendor_id: "10de"
     product_id: "26b9"
     device_type: "type-PF"
+  # Nvidia A30
+  a30:
+    resource_name: "{{ a30_resource_name | default('a30') }}"
+    vendor_id: "10de"
+    product_id: "20b7"
+    device_type: "type-PF"
+  # Nvidia H100L
+  h100l:
```

Comment:
Should this be `h100_nvl` rather than `h100l`? Googling "nvidia h100l" doesn't really come up with anything so maybe `h100l` is a customer-specific name

Reply:
I'd prefer the _nvl suffix just because it's the more common name, but that's just personal preference